src/Nancy.Swagger/Standalone.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Nancy.Swagger.Services;
+using Nancy.TinyIoc;
+using Swagger.ObjectModel;
+
+namespace Nancy.Swagger
+{
+    public static class Standalone
+    {
+        public static SwaggerRoot Generate(TinyIoCContainer container, NancyContext context)
```

Comment:
Did you push your changes?

> Without this class, I wouldn't need to add any code to the library at all, and this could just be a Wiki page.

What's wrong with that?